backend/gameplay/consumers.py

```
@@ -378,7 +389,9 @@ async def update_captured_material(self, board_placement, move_info: dict, en_pa
 		else:
 			chess_game_model.captured_black_material = updated_captured_material
 
-	async def update_position(self, chess_game_model: ChessGame, move_info: dict):
+	async def update_position(self, move_info: dict):
```

Pylint (3.3.7)

[refactor] Too many local variables (23/15)

(R0914)